scienceasia.csl

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" page-range-format="minimal" demote-non-dropping-particle="sort-only" default-locale="en-GB" xmlns="http://purl.org/net/xbiblio/csl">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>ScienceAsia</title>
+    <title-short>ScienceAsia</title-short>
```

Comment:
`<title-short>ScienceAsia</title-short>` is rather redundant, since it's the same as the title. Not a real problem, but it could be deleted.